plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -789,8 +765,8 @@
     "sliceByteString": {
         "cpu": {
             "arguments": {
-                "intercept": 265318,
-                "slope": 0
+                "intercept": 266994,
+                "slope": 11
```

Comment:
Wait, it's a constant function, no? Where does the linearity come from? We have

```haskell
sliceByteStringDenotation start n xs = BS.take n (BS.drop start xs)
```

and both `BS.take` and `BS.drop` are O(1) according to their docs. As well as `BS.length` in the `ExMemoryUsage`.

Reply:
See [this comment](https://github.com/IntersectMBO/plutus/blob/a79131eecfec1292b37eb2cce1e6299b46f360cc/plutus-core/cost-model/create-cost-model/CreateBuiltinCostModel.hs#L475) in the old code and [this one](https://github.com/IntersectMBO/plutus/blob/fe35887112a04ca44f4318b1e526766c2717dad8/plutus-core/cost-model/data/models.R#L497) in the new code.  The slope here is negligible, so the function's essentially constant.  I did wonder whether just to make it actually constant, but that would make the costing function have a less general type than in previous versions.  Maybe that's OK.

Reply:
I see, thanks.